tests/apollo/CMakeLists.txt

```diff
@@ -97,9 +97,11 @@ add_test(NAME skvbc_backup_restore COMMAND sh -c
         "env ${APOLLO_TEST_ENV} BUILD_COMM_TCP_TLS=${BUILD_COMM_TCP_TLS} TEST_NAME=skvbc_backup_restore python3 -m unittest test_skvbc_backup_restore 2>&1 > /dev/null"
         WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
 
-add_test(NAME skvbc_reconfiguration COMMAND sh -c
-        "env ${APOLLO_TEST_ENV} BUILD_COMM_TCP_TLS=${BUILD_COMM_TCP_TLS} TEST_NAME=skvbc_reconfiguration python3 -m unittest test_skvbc_reconfiguration ${TEST_OUTPUT}"
-        WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
+if (USE_S3_OBJECT_STORE)
```

Comment:
Why do we only do it if **USE_S3_OBJECT_STORE** is set?

Reply:
Now the reconfiguration tests contain also pruning with RO replica which requires S3

Reply:
Is there a way to lift this restriction so that whoever builds without can still test reconfiguration?

Reply:
we can separate it into a different test file. For now I will leave it as it is, if we see that it causes issue I will consider to separate it